docs/manage/device.rst

```diff
@@ -16,9 +16,11 @@ Assign Permissions
 
 You can assign additional permissions to Kolibri users which will provide them access to more features compared to their :ref:`user roles <user_roles>`. To manage permissions for Kolibri users, use the **Permission** tab in the  **Device** dashboard (|permissions| icon).
 
-	.. figure:: img/manage-permissions.png
+	.. figure:: ../img/manage-permissions.png
```

Comment:
You can make image references more robust by using absolute paths:

```suggestion
	.. figure:: /img/manage-permissions.png
```

Alternatively, we might try putting images in the same directories as the files that reference them, which I think would allow us to write simply:

```
	.. figure:: manage-permissions.png
```

Reply:
Absolute paths for images is a good idea, but while it's manageable, I prefer keeping them all in the same folder. Keeps the structure cleaner, plus it's easier to review for the optimization step.